Description: build on clean deb13
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 postfixadmin (3.3.10-2) unstable; urgency=medium
 .
   * remove unnessesary fix for mysqli instead of mysql
   * fix permissions of templates_c directory
Author: Christoph Martin <martin@uni-mainz.de>

---
The information above should follow the Patch Tagging Guidelines, please
checkout https://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: (upstream|backport|vendor|other), (<patch-url>|commit:<commit-id>)
Bug: <upstream-bugtracker-url>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: (no|not-needed|<patch-forwarded-url>)
Applied-Upstream: <version>, (<commit-url>|commit:<commid-id>)
Reviewed-By: <name and email of someone who approved/reviewed the patch>
Last-Update: 2025-12-31

--- /dev/null
+++ postfixadmin-3.3.10/ADDITIONS/squirrelmail-plugin/debian/README.Debian
@@ -0,0 +1,20 @@
+Squirrelmail PostfixAdmin Plugin for Debian
+===========================================
+
+After installing the package, you will need to :
+
+1) Edit the config.inc.php file to point to the PostfixAdmin server.
+
+2) Ensure the xmlrpc interface is available and enabled on the Postfixadmin server
+
+3) Run the squirrelmail-configure script.
+
+Where to get help
+=================
+
+See http://squirrelmail-postfixadmin.palepurple.co.uk
+
+Try also : david [at] pale purple dot co dot uk
+
+Or #postfixadmin on irc.freenode.net might be a good bet.
+
--- /dev/null
+++ postfixadmin-3.3.10/ADDITIONS/squirrelmail-plugin/debian/changelog
@@ -0,0 +1,27 @@
+squirrelmail-postfixadmin (2.3.0) stable; urgency=low
+
+  * Using XMLRPC backend (no SQL here)
+
+ -- David Goodwin <david.goodwin@palepurple.co.uk>  Mon, 01 Feb 2010 09:56:00 +0000
+
+squirrelmail-postfixadmin (2.2.0) stable; urgency=low
+
+  * Changed DB backend to use prepared statements
+  * Changed vacation handling to match that of Postfixadmin (vacation.active
+    etc)
+  * Changed vacation page to support UTF8
+
+ -- David Goodwin <david.goodwin@palepurple.co.uk>  Wed, 20 Aug 2008 15:25:00 +0000
+
+squirrelmail-postfixadmin (2.1.1-1) stable; urgency=low
+
+  * Add NL language support
+  * Better db error logging (e.g. if wrong mdb2 driver specified etc)
+
+ -- David Goodwin <david.goodwin@palepurple.co.uk>  Wed, 12 Dec 2007 16:00:00 +0000
+
+squirrelmail-postfixadmin (2.1.0-1) stable; urgency=low
+
+  * Initial release.
+
+ -- David Goodwin <david.goodwin@palepurple.co.uk>  Thu,  8 Nov 2007 20:00:00 +0000
--- /dev/null
+++ postfixadmin-3.3.10/ADDITIONS/squirrelmail-plugin/debian/conffiles
@@ -0,0 +1 @@
+/etc/squirrelmail/plugins/postfixadmin-config.php
--- /dev/null
+++ postfixadmin-3.3.10/ADDITIONS/squirrelmail-plugin/debian/control
@@ -0,0 +1,19 @@
+Source: squirrelmail-postfixadmin
+Section: mail
+Priority: optional
+Maintainer: David Goodwin <david.goodwin@palepurple.co.uk>
+Standards-Version: 3.6.1
+
+Package: squirrelmail-postfixadmin
+Architecture: all
+Depends: squirrelmail, php-pear
+Suggests: postfixadmin
+Description: Plugin for Squirrelmail to integrate with Postfixadmin
+ Postfixadmin is a web based interface for managing mail domains 
+ and users. This package integrates Squirrelmail with it. 
+ Users can change their password, forwarding and vacation settings
+ from within Squirrelmail when this package is installed, and 
+ enabled through the ./squirrelmail-configure command.
+ .
+ For further information see 
+ http://squirrelmail-postfixadmin.palepurple.co.uk
--- /dev/null
+++ postfixadmin-3.3.10/ADDITIONS/squirrelmail-plugin/debian/copyright
@@ -0,0 +1,11 @@
+This package was debianized by David Goodwin <david@palepurple.co.uk>
+2007/11/08
+
+It was downloaded from: http://squirremail-postfixadmin.palepurple.co.uk
+
+Upstream Author(s): n/a
+
+Copyright:
+    Copyright (C) 2007+ by David Goodwin <david@palepurple.co.uk>
+
+License: GPL v2+
--- /dev/null
+++ postfixadmin-3.3.10/ADDITIONS/squirrelmail-plugin/debian/docs
@@ -0,0 +1,4 @@
+LICENSE.txt
+README
+INSTALL
+debian/README.Debian
--- /dev/null
+++ postfixadmin-3.3.10/ADDITIONS/squirrelmail-plugin/debian/postfixadmin-squirrelmail.dirs
@@ -0,0 +1,3 @@
+usr/share/squirrelmail/plugins/postfixadmin
+usr/share/doc/squirrelmail-postfixadmin
+etc/squirrelmail/plugins
--- /dev/null
+++ postfixadmin-3.3.10/ADDITIONS/squirrelmail-plugin/debian/postinst
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+echo "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+echo "WARNING: You need to read /usr/share/doc/squirrelmail-postfixadmin/README.Debian!"
+echo "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
--- /dev/null
+++ postfixadmin-3.3.10/ADDITIONS/squirrelmail-plugin/debian/rules
@@ -0,0 +1,56 @@
+#!/usr/bin/make -f
+# debian/rules makefile for squirrelmail
+
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+
+build: build-stamp
+build-stamp:
+	dh_testdir
+
+clean:
+	dh_testdir
+	dh_testroot
+	dh_clean
+
+install: build
+	$(checkdir)
+	dh_testdir
+	dh_testroot
+	dh_clean -k
+	dh_installdirs
+	dh_install
+	mkdir -p debian/tmp/usr/share/squirrelmail/plugins/postfixadmin
+	cp -a *.php version debian/tmp/usr/share/squirrelmail/plugins/postfixadmin
+	cp -a locale debian/tmp/usr/share/squirrelmail/plugins/postfixadmin
+	cp -a po debian/tmp/usr/share/squirrelmail/plugins/postfixadmin
+	mkdir -p debian/tmp/etc/squirrelmail/plugins/
+	cp -a *.sample debian/tmp/etc/squirrelmail/plugins/postfixadmin-config.php
+	mkdir -p debian/tmp/DEBIAN
+	cp debian/postinst debian/tmp/DEBIAN/postinst
+	chmod 555 debian/tmp/DEBIAN/postinst
+	ln -s /etc/squirrelmail/plugins/postfixadmin-config.php debian/tmp/usr/share/squirrelmail/plugins/postfixadmin/config.php
+	find debian/tmp -name .svn | xargs -r rm -r
+
+# Build architecture-independent files here.
+binary-indep: build install
+	dh_testdir
+	dh_testroot
+	dh_installdebconf	
+	dh_installdocs -X.svn
+	dh_installexamples
+	dh_installman
+	dh_installcron
+	dh_link
+	dh_compress
+	dh_fixperms -X/var
+	dh_installdeb
+	dh_gencontrol
+	dh_md5sums
+	dh_builddeb
+
+# Build architecture-dependent files here.
+binary-arch:
+
+binary: binary-indep binary-arch
+.PHONY: build clean binary-indep binary-arch binary install
